### PR TITLE
Completed css-two-column-layout exercise.

### PR DESCRIPTION
**Pull request by AnthonyVNgo**
Screen Shot 2022-03-18 at 00 34 40

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <div class=column-one-fourth><img src="images/software.svg" alt=""></div>
+      </div>
+
+      <div class="card">
+        <div class=column-one-fourth><img src="images/software.svg" alt=""></div>
+      </div>
+
+      <div class="card">
+        <div class=column-one-fourth><img src="images/software.svg" alt=""></div>
+      </div> -->
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: Please remove commented code prior to submitting for review.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+.hcenter {
+  justify-content: center;
+}
+
+.txt-center {
+  text-align: center;
+}
+
+.column-one-fourth {
+  align-self: flex-start;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: The only styling that should be applied to a `column` class is either `width` or `flex-basis`. If you need additional styling, it should be applied to a separate class that you can add in addition to the `column` class.

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  flex-basis: 25%;
+}
+
+.column-three-fourths {
+  flex-basis: 75%;
+  padding: 40px 0
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: The only styling that should be applied to a `column` class is either `width` or `flex-basis`. If you need additional styling, it should be applied to a separate class that you can add in addition to the `column` class.

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+  /* align-self: flex-end; */
+
+  /* align-self: center; */
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: Remove commented code, please

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-two-column-layout/styles.css (hunk 4 above)
Comment (AnthonyVNgo) [reply css-two-column-layout/styles.css]: Screen Shot 2022-03-18 at 13 45 51 (2)

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**